app/sequencescape/sequencescape/api/v2/submission_pool.rb

```diff
@@ -1,4 +1,6 @@
 # frozen_string_literal: true
 
+# submission pool resource
 class Sequencescape::Api::V2::SubmissionPool < Sequencescape::Api::V2::Base
+  has_many :tag_layout_templates
```

Comment:
wondering why this has_many was added

Reply:
It's to support this usage of the API: app/models/labware_creators/tagging/tag_collection.rb
Abdullah had created the SubmissionPool endpoint on API v2 at the same time as me and followed the pattern of v1.  Properties (like v1 used) do not need to be declared, but relationships (like I used) do have to be declared in the client.